src/Feedback-Service/Dtos/UpdateFeedbackDto.cs

```diff
@@ -1,8 +1,9 @@
 namespace Feedback_Service.Dtos;
 
-public class UpdateFeedbackDTO
+public class UpdateFeedbackDto
 {
-    public Guid PatientId { get; set; }
+    public int PatientId { get; set; }
+    public Guid AuthorId { get; set; }
     public Guid StressMeassurementId { get; set; }
-    public string Feedback { get; set; } = "";
+    public string? FeedbackComment { get; set; }
```

Comment:
I believe this too could be a record. If that causes any issues, let's have a look together.

Reply:
yesss, just resolved it